### PR TITLE
F #src 1550 add sim ensenso

### PR DESCRIPTION
**Pull request by ethanfowler**

### CODE REVIEW

File: sr_camera_launch/launch/ensenso.launch [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<launch>
+  <arg name="sim" default="true" />
+  <arg name="depth_registration" default="true" />
+  <arg name="camera" default="camera" />
+  <arg name="tf_prefix" default="" />
+  <arg if="$(arg sim)" name="pose_estimation" default="false" />
```
Comment (mikramarc) [inline sr_camera_launch/launch/ensenso.launch]: pose estimation arg is used only in ensenso frames that is used only in sim, so why have this arg in the first place?

**Review by mikramarc — CHANGES_REQUESTED** [review]

File: sr_camera_launch/launch/ensenso.launch (hunk 1 above)
Comment (ethanfowler) [reply sr_camera_launch/launch/ensenso.launch]: Fixed in 2a495205b2f992570b23e967f55d40b6991f281d.

Forgot to pipe depth registration arg through to the hardware launch file (which also publishes TFs dependent on pose estimation).

File: sr_camera_launch/launch/ensenso.launch [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <arg name="sim" value="true"/>
+    <arg name="publish_camera_tf" value="$(arg publish_camera_tf)"/>
+    <arg name="x" value="$(arg x)"/>
+    <arg name="y" value="$(arg y)"/>
+    <arg name="z" value="$(arg z)"/>
+    <arg name="R" value="$(arg R)"/>
+    <arg name="P" value="$(arg P)"/>
+    <arg name="Y" value="$(arg Y)"/>
+  </include>
+
```
Comment (mikramarc) [inline sr_camera_launch/launch/ensenso.launch:32]: ugly

**Review by mikramarc — CHANGES_REQUESTED** [review]

File: sr_camera_launch/launch/ensenso.launch (hunk 2 above)
Comment (ethanfowler) [reply sr_camera_launch/launch/ensenso.launch:32]: Fixed in 060db93e3cf5ab39989c7cb91721ce4cb6660975

**Review by mikramarc — APPROVED** [review]

**Pull request merged**